### PR TITLE
Feature/pelagos 4684 create crud application

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Form/InformationProductType.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            ->add('creationTimeStamp')
+            ->add('modificationTimeStamp')
+//            ->add('researchGroups', CollectionType::class, array(
+//                'label' => 'Research Groups',
+//                'entry_type' => ChoiceType::class,
+//                'entry_options' => array(
+//                    'choices' => ["701", "807"],
+//                ),
+//                'required' => true,
+//            ))
```
Comment (mickel1138) [inline src/Form/InformationProductType.php:35]: Can be removed.

File: templates/InformationProduct/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html.twig'%}
+{% block title %}Information Product{% endblock %}
+
+{%block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('information-product') }}
+
```
Comment (mickel1138) [inline templates/InformationProduct/index.html.twig:7]: Maybe remove?

**Review by fightingtexasaggie — COMMENTED** [review]
I don't see an interface route to edit existing entries. Also, at a minimum, we probably want a simple list view with links to the aforementioned edit route.

**Comment by mickel1138** [comment]
> I don't see an interface route to edit existing entries. Also, at a minimum, we probably want a simple list view with links to the aforementioned edit route.

This will be on a different ticket.

**Review by mickel1138 — APPROVED** [review]
Good for PO review.

**Review by fightingtexasaggie — APPROVED** [review]
A pretty solid first iteration imho.

**Pull request merged**